packages/fe-fpm-writer/src/app/index.ts

```diff
@@ -63,20 +63,41 @@ export function enableFPM(basePath: string, config: FPMConfig = {}, fs?: Editor)
     }
 
     // if a minUI5Version is set and it is smaller than the minimum required, increase it
-    const minUI5Version = manifest['sap.ui5']?.dependencies?.minUI5Version;
-    if (minUI5Version && valid(minUI5Version) && lt(minUI5Version, MIN_VERSION)) {
-        fs.extendJSON(manifestPath, {
-            'sap.ui5': {
-                dependencies: {
-                    minUI5Version: MIN_VERSION
-                }
+    const minUI5VersionArray = getMinUI5VersionAsArray(manifest);
```

Comment:
Later in line 71 we do `valid(minUI5Version)`, but calling `getMinUI5VersionAsArray(manifest)` will never return an invalid version. Should this call be:
```suggestion
    const minUI5VersionArray = getMinUI5VersionAsArray(manifest, true);
```
?

Reply:
Agree, this must change. Done